backend/src/v2/cmd/driver/main.go

```diff
@@ -70,6 +75,11 @@ var (
 	cachedDecisionPath = flag.String("cached_decision_path", "", "Cached Decision output path")
 	conditionPath      = flag.String("condition_path", "", "Condition output path")
 	logLevel           = flag.String("log_level", "1", "The verbosity level to log.")
+
+	// proxy
+	httpProxy  = flag.String(httpProxyArg, unsetProxyArgValue, "The proxy for HTTP connections.")
```

Comment:
From my reading of the code, the Driver itself does not use these but uses it as environment variables for the Launcher Pod in the Pod spec patch. Should Driver itself uses these by setting these environment variables after validating the flag values?

I think the only connections it makes are to MLMD and the Kubernetes API but it could be good to set this to be on the safe side.

Reply:
I'm not sure it's needed. But I agree on adding it for safety.